Hw1 - Updated H1-newBranch

lee-112 opened the pull request:
Updated code changes made to H1 new branch:
1. `.gitignore` Updated ignore files/packages for ESLinter Airbnb setup.
2. `01-selectors.md` Added solutions for CSS Dinner Game.
3. `palindrome.html` Added Bootstrap 5 styling
4. `palindrome.js` Added functionality and Eventlisteners for getting and setting DOM content.

anhoop89 commented on hw1/02-palindrome.js:
```diff
@@ -1,3 +1,28 @@
-const elem = document.querySelector('input');
+document.addEventListener('DOMContentLoaded', () => {
```
I really like this way to check a palindrome. I've learned from you. 
Since it's a short code, it won't take too long to understand what you did. A little node I would say that you should leave a few lines of comments on what you did. That would be great!

anhoop89 commented on hw1/02-palindrome.js:
```diff
@@ -1,3 +1,28 @@
+      if (Number.isSafeInteger(numberInt) && numberInt >= 0) {
+        const isNumberPalindrome = isPalindrome(numberInt);
+        outputElement.innerHTML = isNumberPalindrome
+          ? '<p class="m-0" style="color: Green;">Yes, This is a Palindrome!</p>'
```
since she wants us to review the classes in bootstrap, I found out: 
Red
.text-success
Green
.text-danger
Yellow
.text-warning

anhoop89 commented on hw1/01-selectors.md:
```diff
@@ -2,6 +2,36 @@
+8. bento orange.small
+
+9. plate, bento
+10. \*
```
i think it should be * only. 
I tried "\\*" that didn't work.

lee-112 replied:
For Question 10, it is essential to include a forward-slash `/` before the asterisk `*` symbol to ensure it is displayed correctly. When you access your hw1 repository and examine the 01-selector.md file, you will notice that the `*` symbol is not visible in your answer for Question 10. By default, GitHub presents all .md files in the markdown view. To confirm that you have inserted an asterisk, you would need to click the 'Raw' button to view the markdown source code.

anhoop89 reviewed (commented):
I have reviewed the changes you made and to be honest, they look good. Your solution is great and I have learned from you as well. There are just a couple of minor things that I mentioned in each file. Overall, good job!

hollan2 reviewed (commented):
Ran your code and everything looks great! Having the palindrome results tied to the enter button took me a little while to figure out because I thought it was supposed to be tied to the input instead. I read the homework instructions and it seems like having it tied to the enter button is fine though.

lee-112 commented:
I have made some updated code changes to the palindrome.html and palindrome.js files. 
1. Added a `<style>` tag to palindrome.html 
2. Added class names `text-green` and `text-red` to palindrome.js

These two code changes were made under the guidance of Professor Caterina Paun to fix the color contrast error from the Wave accessibility test.
@hollan2 @anhoop89
